packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -115,13 +119,22 @@ export default {
     },
 
     methods: {
+        ...mapActions('checkout', [
+            'updateFulfilmentAddress'
+        ]),
         /*
         * Returns true if `field` has been touched and if it is still empty
         * The $dirty boolean changes to true when the user has focused/lost
         * focus on the input field.
         */
         isFieldEmpty (field) {
             return this.$v.addressValidations[field].$dirty && !this.$v.addressValidations[field].required;
+        },
+        /*
+        * Dispatches map action `updateFulfilmentAddress` to update input fields values in vuex
+        */
+        updateAddressDetails (key, event) {
+            this.updateFulfilmentAddress({ [key]: event })
```

Comment:
Will this not pass the whole event object now? I think you will need to supply the value here from the event possibility?

Reply:
The event in this case is a string because form-field components [emits the input](https://github.com/justeat/fozzie-components/blob/cae0417e2cc188b47a30a0176b3d491aa7d0cf12/packages/components/atoms/f-form-field/src/components/FormField.vue#L216) event and passes along only the value:
```
updateValue (event) {
    this.$emit('input', event.target.value);
}
```

Reply:
@zlatinivanov-je because of the above :point_up_2: your suggestion unfortunately wouldn't work

Reply:
Is it worth renaming this param to `value` then? Just so it's a little clear that the input value is being passed along like this? so instead of `[key]: event` it can be `[key]: value`

Reply:
Sure, will do it now